source/Frame.lagda

```diff
@@ -495,6 +541,31 @@ syntax binary-join F x y = x ∨[ F ] y
   γ = z                      ≤⟨ ∨[ F ]-upper₂ y z            ⟩
       y ∨[ F ] z             ≤⟨ ∨[ F ]-upper₂ x (y ∨[ F ] z) ⟩
       x ∨[ F ] (y ∨[ F ] z)  ■
+
+∨[_]-left-mono : (F : frame 𝓤 𝓥 𝓦)
```

Comment:
"mono" is confusing (a map is called a mono in category theory, which different from monotone, which is what you intend here).

Reply:
Done ✅